src/pages/dashboard/workplace/components/EditableLinkGroup/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { PureComponent, createElement } from 'react';
+import { PlusOutlined } from '@ant-design/icons';
+import { Button } from 'antd';
+import styles from './index.less';
+
+export interface EditableLink {
+  title: string;
+  href: string;
+  id?: string;
+}
+
+interface EditableLinkGroupProps {
+  onAdd: () => void;
+  links: EditableLink[];
+  linkElement: React.ComponentClass<any>;
+}
+
+class EditableLinkGroup extends PureComponent<EditableLinkGroupProps> {
+  static defaultProps = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected member-variable-declaration: 'defaultProps' to have a typedef](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)